spring-cloud-deployer-cloudfoundry/src/main/java/org/springframework/cloud/deployer/spi/cloudfoundry/ApplicationLogAccessor.java

```diff
@@ -75,8 +75,15 @@ public String getLog(String deploymentId, Duration apiTimeout) {
             stringBuilder.append(log.getPayloadAsText());
             stringBuilder.append(System.lineSeparator());
         });
-
-        return stringBuilder.toString();
+        String [] lines = stringBuilder.toString().split("\n");
```

Comment:
Lines 74+ can be replaced w/ these: 
```java
        Collections.reverse(logs);
        return logs.stream().map(Log::getPayloadAsText)
                .collect(Collectors.joining(System.lineSeparator()));
```
😄

Reply:
This change did not work :-(